.changeset/sharp-planets-dress.md

```diff
@@ -0,0 +1,5 @@
+---
+'@microsoft/atlas-css': patch
+---
+
+Adding control-theme to the theme palette and applying it to the form controls; fixing select component font size.
```

Comment:
color-border, not control-theme

Reply:
🤦🏻‍♀️

Reply:
Adding control-border to the theme palette and apply it to the form controls; fix select component font size.

Reply:
Different changeset: " fix select component font size."

Reply:
Oh, I didn't know I can do it several times in the same PR. I'll update

Reply:
Updated